site/docs/get-started/quickstart.md

```diff
@@ -0,0 +1,70 @@
+# Quick Start Guide
+
+Follow this guide to install Mesh for Data using default parameters that are suitable for experimentation.
+<!-- For a full installation refer to the [full installation guide](./setup/install) instead. -->
+
+## Before you begin
+
+Ensure that you have the following:
+
+- [Helm](https://helm.sh/) 3.3 or newer must be installed and configured on your machine.
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.16 or newer must be installed on your machine.
+- Access to a Kubernetes cluster such as [Kind](http://kind.sigs.k8s.io/) as a cluster administrator.
+
+## Install cert-manager
+
+Mesh for Data requires [cert-manager](https://cert-manager.io) to be installed to your cluster. 
+Many clusters already include cert-manager. Run the following to install cert-manager only if it's missing:
+
+```bash
+kubectl get namespace cert-manager || kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.2.0/cert-manager.yaml
```

Comment:
We have three different ways now that explain or do install the cert-manager. This here (with v1.2.0). In the third-party folder (with v0.13.1) and in the charts folder via helm chart (with v1.1.0). Also the yaml you referenced contains an item for the `cert-manager` namespace. 
We should remove parts that we don't want and agree on which version to use. (And I'm not sure if we should reference that openshift has yet another way to install this)

Reply:
Right. The one in third_party is obselete and should be removed but it's unrelated to this PR.

I will need to check their Helm chart and change here to be closer to our Makefile. I opted for the manifests because their instructions are the same for Kubernetes and OpenShift. Will check the chart and change accordingly. 

> Also the yaml you referenced contains an item for the cert-manager namespace.

I'm aware. What the instructions here do is to install cert-manager only if the namespace doesn't already exist. This avoids messing up cert-manager in clusters that already have a version of it installed. I'll make this more explicit along with the other change.